catalog/imaging/imaging-transformer-nitf/src/main/java/org/codice/alliance/transformer/nitf/gmti/GmtiMetacardType.java

```diff
@@ -58,5 +59,6 @@ public void initDescriptors() {
         descriptors.addAll(getDescriptors(CsdidaAttribute.getAttributes()));
         descriptors.addAll(getDescriptors(CsexraAttribute.getAttributes()));
         descriptors.addAll(getDescriptors(HistoaAttribute.getAttributes()));
+        descriptors.addAll(getDescriptors(PiatgbAttribute.getAttributes()));
```

Comment:
I was able to refactor some of this code to add the common attribute descriptors between the ImageMetcardType and the GmtiMetacardType into the AbstractNitfMetacardType. Any specific attribute descriptors should be added into the corresponding metacard type. 

@bdeining As far as doing this in blueprint, adding TRE attributes descriptors to a metacard type works differently than adding attribute descriptors that come from (TopicAttributes.java as an example) so it will not work the same way as the example you have provided.